xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_version.xsd

```diff
@@ -199,6 +200,11 @@ Rail transport, Roads and Road transport
 				</xsd:annotation>
 			</xsd:element>
 			<xsd:element ref="TransportTypeRef" minOccurs="0"/>
+			<xsd:element name="FloatingVehicles" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The specific ZONE where a vehicle must be collected/returned from a station rather than anywhere in zone. +v2.0</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Tells whether in this ZONE a vehicle must be collected/returned from a station rather than anywhere in zone. +v2.0</xsd:documentation>
```